docs/tutorials/using-image-generation-models.md

```diff
@@ -47,6 +47,288 @@ The generated image will be displayed in the UI. Your image may look different g
 
 ![Generated](../assets/tutorials/using-image-generation-models/image-playground.png)
 
+## Extensions
+
+GPUStack currently does not support GGUF image models that are not all-in-one (base model, text encoder, and VAE combined). Please refer to the supported model list for details:
+
+● [Hugging Face](https://huggingface.co/collections/gpustack/image-672dafeb2fa0d02dbe2539a9)
+
+● [Model Scope](https://modelscope.cn/collections/Image-fab3d241f8a641)
+
+### Basic Configuration Guide
+
+The core parameters of image generation models are key to achieving desired outputs. These include `Prompt`, `Seed`, `Resolution`, `Sampler`, `Scheduler`, `Sampling Steps` and `CFG scale`.
+Different models may have variations in parameter settings. In order to quickly get started and generate satisfying images, the following section is going to provide some reference values for parameter configurations.
+
+#### FLUX.1-dev
```

Comment:
The tutorial deploys sd-v3-5-medium only. Should we put the parameter guidance to a separate page, and linked it in the end of this tutorial for reference?

Reply:
Updated